services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/unit_templates_manager/unitTemplateMinisTable.tsx

```diff
@@ -58,22 +65,44 @@ export const UnitTemplateMinisTable = ({ unitTemplates, userSignedIn }) => {
     )
   }
 
+  function getNameElement(name: string, activities: Activity[]) {
+    return(
+      <Tooltip
+        tooltipText={renderActivityPackTooltipElement({ activities })}
+        tooltipTriggerText={name}
+        tooltipTriggerTextClass="activity-pack-name"
+      />
+    )
+  }
+
+  function getChronologicalGradeLevel(readability: string) {
+    if(!readability) { return 0 }
+    if(readability.slice(0,2) === '10') { return 10 }
```

Comment:
I probably should have caught this in an earlier PR, but this is probably supposed to be the actual grade level, not readability, right? Or no?